src/controllers/sdex/index.ts

```diff
@@ -29,12 +39,29 @@ router.get(
   }),
 );
 
-router.get('/user_pool_positions', async (req, res) => {
-  res.status(200).json({
-    data: {
-      user_pool_positions: [],
-    },
-  });
-});
+router.get(
+  '/user_pool_positions',
+  asyncRoute(async (req, res) => {
+    const { error, value } = querySchema.validate(req.query);
+
+    if (error) {
+      throw new BadRequestError(error.details[0].message);
+    }
+
+    const { user, chainId, base, quote, poolIdx } = value;
```

Comment:
Use `validate` helper from `utils/validation.ts`, it will handle error rendering for you.

```suggestion
    const { user, chainId, base, quote, poolIdx } = validate(querySchema, req.query);
```